source/recommended-reading.md

```diff
@@ -24,35 +24,47 @@
 
 ## Other articles
 
-- [NixOS and Flakes - An unofficial book for beginners](https://nixos-and-flakes.thiscute.world) (2023)
+- [Nix Pills](https://nixos.org/nixos/nix-pills/index.html)
 
-  This tutorial is an introduction to {term}`NixOS` using the experimental Nix [Flakes](flakes) functionality.
+  A low-level tutorial on building software packages with Nix, showing in detail how Nixpkgs is constructed.
 
 - [Customizing packages in Nix](https://bobvanderlinden.me/customizing-packages-in-nix/) (2022)
 
-  An overview of different methods to customize Nix packages.
-
 - [Manage your dot files with Home Manager](https://ghedam.at/24353/tutorial-getting-started-with-home-manager-for-nix) (Mattia Gheda, 2021)
 
-- [Nix Pills](https://nixos.org/nixos/nix-pills/index.html)
-
-  A low-level tutorial on building software packages with Nix, showing in detail how Nixpkgs is constructed.
+  An overview of different methods to customize Nix packages.
 
 - [Nix Shorts](https://github.com/justinwoo/nix-shorts)
 
   A series of posts on basics of how packaging with Nix works.
 
-- [awesome-nix](https://nix-community.github.io/awesome-nix/)
```

Comment:
Why remove awesome-nix?

Reply:
Because it's already in the side bar.

Reply:
Oh I see, it wasn't obvious from the title "Support tools" :)